src/web/plot.rkt

```diff
@@ -243,25 +239,30 @@
              (for/list ([i (in-range idx-min (+ idx-min bin-size))]) (vector-ref errs i))])
            <)))))
 
-(define (error-avg errs pts #:axis [axis 0] #:vars [vars '()]
+(define (error-avg errs pts repr #:axis [axis 0] #:vars [vars '()]
                    #:color [color *blue-theme*] #:bin-size [bin-size 128])
   (define get-coord
     (if (number? axis)
         (λ x (list-ref x axis))
         (eval-prog `(λ ,vars ,axis) 'fl)))
+
+   ; works for binary64, binary32 (probably not for posits)
+  (define-values (maxbound minbound)
+    (let ([hi (sub1 ((representation-repr->ordinal repr) (fl->repr +inf.0 repr)))]
+          [lo (add1 ((representation-repr->ordinal repr) (fl->repr -inf.0 repr)))])
+      (values ((representation-ordinal->repr repr) hi)
+              ((representation-ordinal->repr repr) lo))))
```

Comment:
I think this is good.

Reply:
We assume ordinals are sorted, but you're right, I don't really know what this does given that posits have only one infinity. Hmm. Let's leave it for now.